src/modules/APIs.js

```diff
@@ -0,0 +1,54 @@
+// I run this code in console when I want to reset the board by creating a id for involvement api
+
+// fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/', {
+//   method: 'POST',
+//   body: JSON.stringify({
+//     name: 'comment',
+//   }),
+//   headers: {
+//     'Content-type': 'application/json; charset=UTF-8',
+//   },
+// })
+//   .then((response) => response.json())
+//   .then((json) => console.log(json));
+
+// the id I get running the previous fetch code
+// const ID = 'woTq8RsLkSavLaCMQAAP';
```

Comment:
- [ ] Please remove any unused code from your codebase for a cleaner and more optimized environment. 🙏 see more tips [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=Do%20not%20commit%20old%20pieces%20of%20code%20as%20inline%20comments.).